bin/varnishd/http2/cache_http2_panic.c

```diff
@@ -49,21 +72,53 @@ h2_sess_panic(struct vsb *vsb, const struct sess *sp)
 	h2 = (void*)*up;
 	if (PAN_dump_struct(vsb, h2, H2_SESS_MAGIC, "h2_sess"))
 		return;
-	h2 = (void*)*up;
+	VSB_printf(vsb, "refcnt = %d, bogosity = %d, error = %s\n",
+	    h2->refcnt, h2->bogosity, h2_panic_error(h2->error));
+	VSB_printf(vsb,
+	    "open_streams = %d, highest_stream = %u,"
+	    " goaway_last_stream = %u,\n",
+	    h2->open_streams, h2->highest_stream, h2->goaway_last_stream);
```

Comment:
h2->open_streams is unsigned, should use %u

Reply:
Fixed